### PR TITLE
Fix deploy and configure cross-origin requests

### PR DESCRIPTION
**Pull request by maxim12321**

### CODE REVIEW

File: src/main/resources/application-release.properties [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # Database credentials
-spring.datasource.url=${CLEARDB_DATABASE_URL}
-spring.datasource.username=${CLEARDB_DATABASE_USERNAME}
-spring.datasource.password=${CLEARDB_DATABASE_PASSWORD}
+spring.datasource.url=jdbc:${AWS_DATABASE_URL}
+spring.datasource.username=${AWS_DATABASE_USERNAME}
+spring.datasource.password=${AWS_DATABASE_PASSWORD}
+
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
```
Comment (AntonMaksimenkau) [inline src/main/resources/application-release.properties]: should we really specify driver and dialect? I think it can be found automatically from the url

**Review by AntonMaksimenkau — DISMISSED** [review]

**Review by Darina007 — DISMISSED** [review]

File: src/main/resources/db/changelog/changes/schema.xml [hunk 2]
```diff
@@ -184,6 +184,7 @@
     </changeSet>
 
     <changeSet id="update_status_type" author="Anlord241">
+        <validCheckSum>8:429e8ab296a5ea20f0e1bb7bf7a5f389</validCheckSum>
```
Comment (AntonMaksimenkau) [inline src/main/resources/db/changelog/changes/schema.xml:187]: is it fix for updated DB? Will it work with a new one?

**Review by AntonMaksimenkau — APPROVED** [review]

File: src/main/resources/db/changelog/changes/schema.xml (hunk 2 above)
Comment (maxim12321) [reply src/main/resources/db/changelog/changes/schema.xml:187]: This tag adds one more valid checksum, so both old and new chageSets will be valid.

Comment (maxim12321) [reply src/main/resources/db/changelog/changes/schema.xml:187]: So, for a new one, it will just execute changeSet. This fix is only for updated DB, yep

**Review by Darina007 — APPROVED** [review]

**Pull request merged**